documentation/docusaurus.config.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
   projectName: 'reveal',
   themeConfig: {
     algolia: {
+      appId: 'reveal-docs',
```

Comment:
Did we add this `appId` in the `algolia` for Reveal account or do not know if it is actually needed?

Reply:
I got an error when it wasn't included in the config.. So I just added this, and it seems to work.  I have no idea if it can be an arbitrary name, or if it should match something

Reply:
Well, apparently it should be set to an ID we have received from Algolia: https://docusaurus.io/docs/search But I don't think any of the current team members have an idea of what it should be